baklava/package.json

```diff
@@ -68,6 +68,21 @@
                 "deb",
                 "AppImage",
                 "snap"
+            ],
+            "category": "AudioVideo"
+        },
+        "snap": {
+            "confinement": "strict",
+            "plugs": [
+                "default",
+                "audio-playback",
+                "audio-record",
```

Comment:
have you properly tested this locally? does it fix the mic problems?

Reply:
I'll try to do the build first, the install my build with the `--dangerous` flag (`snap install baklava/build<dogehouse-build-i-reproduced>.snap --dangerous` and check if it's work.

Because the `audio-record` interface isn't automatically connected due to privacy reasons as per the Snapcraft docs, I may manually connect with `snap connect dogehouse:audio-record` and test if it's detected.

Reply:
yes please test it out.

Reply:
@amitojsingh366 It works! (ref: https://open.lbry.com/screencast-devlog-1:b70f45420143972e1f6ea014c4a80cc17194911f)